### PR TITLE
Support easylogging config for segcore and knowhere

### PR DESCRIPTION
**Pull request by cydrain**
Signed-off-by: yudong.cai <yudong.cai@zilliz.com>
Issue: #16732

### CODE REVIEW

**Comment by jennyli-z** [comment]
rerun ut

File: configs/segcore.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+* GLOBAL:
```
Comment (xiaofan-luan) [inline configs/segcore.yaml:1]: why is segcore not follow style of milvus.yaml?

Comment (cydrain) [reply configs/segcore.yaml:1]: this yaml is for easylogging, easylogging has special request for yaml format.

**Pull request merged**